File: CodingTest/greedy/greedyAlgorithm.js
"use strict";
let check = 100000;
let payPrice = 34830;
let restMoney = 65170; //15170 
let money = 50000;
let money2 = 10000;
function change_moneys(a, b) {
    const num = a % b;
    if (num === 0) {
        console.log(false);
        // 50000 * 나머지(정수)
        // restMoney - 50000 
        // 결과 값 출력 (15710) 
    }
    else {
        console.log(true);
        let restMoney2 = Math.floor(num);
        console.log(restMoney2);
        let temp = restMoney;
        restMoney = restMoney2;
        restMoney2 = restMoney;
        money = money2;
        money2 = money;
        console.log(restMoney, money);
    }
}
change_moneys(restMoney, money);

File: CodingTest/greedy/greedyAlgorithm.py
import math

check = 100000
payPrice = 34830
restMoney = 65170;
money = 50000;
money2 = 10000; 




def change_money(a, b) :
    num = a % b
    if num == 0 :
        print('false')
      
    else:
        print('true')
        restMoney2 = math.floor(num);
        print(restMoney2)
        restMoney = 65170;
        money = 50000;
        money2 = 10000; 
        temp = restMoney
        restMoney = restMoney2
        restMoney2 = restMoney
        money = money2
        money2 = money
        print(restMoney, money)
      


change_money(restMoney, money)
